index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body class="page">
+    <header class="header">
+      <nav>
+        <img src="images/menu.svg" alt="Menu">
+      </nav>
+    </header>
+    <section>
+        <div class="headline">
+          <span class="background-image">
+            <!-- <img src="images/header.svg" alt=""> -->
```

Comment:
- [optional] kindly remove the commented piece of code for cleaner code practice